content/source/docs/enterprise/sentinel/import/mock-tfconfig.html.md

```diff
@@ -22,9 +22,11 @@ documentation][ref-official-sentinel-documentation].
 [ref-mocking-imports]: https://docs.hashicorp.com/sentinel/writing/imports#mocking-imports
 [ref-official-sentinel-documentation]: https://docs.hashicorp.com/sentinel/
 
--> **NOTE:** As functions cannot be mocked in the current Sentinel testing
-framework, the [module()][ref-module] function is not available. As a result,
-only root module data can be mocked at this time.
+#### Current mock limitations
+
+As functions cannot be mocked in the current Sentinel testing framework, the
```

Comment:
```suggestion
* As functions cannot be mocked in the current Sentinel testing framework, the
```

Reply:
Changing from NOTEs here to headings makes it kind of look as if the mock itself has moved down a level. I think the list format helps counteract that, so even though there's only one bullet here, I'd suggest sticking with that convention.